app/services/data/get-reductions.js

```diff
@@ -0,0 +1,25 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+
+module.exports = function (id) {
```

Comment:
Could you call this variable workloadOwnerId?  Means we don't get confused about whether it is the id of the actual reduction

Reply:
Done.